### PR TITLE
fix ocr3 int test

### PR DESCRIPTION
**Pull request by makramkd**

### CODE REVIEW

File: core/services/ccipcapability/configs/evm/contract_reader.go [hunk 1]
```diff
@@ -150,6 +140,45 @@ func SourceReaderConfig() evmrelaytypes.ChainReaderConfig {
 					},
 				},
 			},
+			consts.ContractNamePriceRegistry: {
+				ContractABI: price_registry.PriceRegistryABI,
+				Configs: map[string]*evmrelaytypes.ChainReaderDefinition{
+					// TODO: update with the consts from https://github.com/smartcontractkit/chainlink-ccip/pull/39
```
Comment (elatoskinas) [inline core/services/ccipcapability/configs/evm/contract_reader.go:146]: q: are all external functions needed (as in an ABI definition), or only the external ones? `ProcessMessageArgs`, `ValidatePoolReturnData` and `GetValidatedTokenPrice` would not be needed for configs

File: core/services/ocr3/plugins/ccip_integration_tests/helpers.go [hunk 2]
```diff
@@ -115,7 +115,7 @@ func createUniverses(
 		rmnProxy := deployARMProxyContract(t, owner, backend, rmn.Address(), chainID)
 		weth := deployWETHContract(t, owner, backend, chainID)
 		rout := deployRouter(t, owner, backend, weth.Address(), rmnProxy.Address(), chainID)
-		priceRegistry := deployPriceRegistry(t, owner, backend, linkToken.Address(), weth.Address(), chainID)
+		priceRegistry := deployPriceRegistry(t, owner, backend, linkToken.Address(), weth.Address(), big.NewInt(1e18), chainID)
```
Comment (elatoskinas) [inline core/services/ocr3/plugins/ccip_integration_tests/helpers.go:118]: suggestion: should we separate `big.NewInt(1e18)` into a const? Not clear what this is without a named variable

File: core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go [hunk 3]
```diff
@@ -173,10 +173,12 @@ func TestIntegration_OCR3Nodes(t *testing.T) {
 			paddedAddr := common.LeftPadBytes(chainPingPongAddr, len(log.Message.Receiver))
 			require.Equal(t, paddedAddr, log.Message.Receiver)
 
-			// check that sequence number is bumped in the onramp.
-			dcc, err = uni.onramp.GetDestChainConfig(&bind.CallOpts{}, log.DestChainSelector)
+			// check that sequence number is equal to the expected next sequence number.
+			// and that the sequence number is bumped in the onramp.
+			require.Equalf(t, log.Message.Header.SequenceNumber, expNextSeqNr, "incorrect sequence number in CCIPSendRequested event on chain %d", log.DestChainSelector)
+			newExpNextSeqNr, err := uni.onramp.GetExpectedNextSequenceNumber(&bind.CallOpts{}, getSelector(otherChain))
```
Comment (elatoskinas) [inline core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go:179]: q: is there any reason we needed `getExpectedNextSequenceNumber` off-chain (for simplicity reasons?) We could just change it to `getSequenceNumber` to not require any +1 / -1 handling

**Review by elatoskinas — APPROVED** [review]

File: core/services/ccipcapability/configs/evm/contract_reader.go (hunk 1 above)
Comment (makramkd) [reply core/services/ccipcapability/configs/evm/contract_reader.go:146]: Its not clear what we'd need offchain for now, I thought we may need GetValidatedTokenPrice at least to do ARL calculations in exec.

Comment (makramkd) [reply core/services/ccipcapability/configs/evm/contract_reader.go:146]: My thinking was we'd get everything working and come back here to trim anything not needed.

File: core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go (hunk 3 above)
Comment (makramkd) [reply core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go:179]: That's a good question, I'm not sure it makes a huge difference if its next expected vs. seq num, you'd either have to +1 or -1 depending on the exact use case.

**Pull request merged**